ui/ts/util/format.ts

```diff
@@ -1,4 +1,5 @@
 // source: util/format.ts
+/// <reference path="../typings/d3/d3.d.ts" />
```

Comment:
Hmm, I was actually looking at the wrong reference to `d3.d.ts`. There was indeed a superfluous reference to it in status.ts (also in this code review), but it wasn't added in this commit.

This reference is apparently not needed when compiling the entire project; however, it IS needed if you compile this individual file, which vim's typescript plugin does. This file does reference d3, so it is appropriate to reference those typings.